.gitignore

```diff
@@ -27,4 +26,4 @@ tutorials/recordings
 tutorials/cursor_image.png
 __pycache__/
 instance/
-mslib/idp/modules
+mslib/msidp/modules
```

Comment:
@nilupulmanodya what is the reason to have this there. I mean, if we have installed MSS and we would change content inside the installed dir, likly installmanagers will run into problems, because the state of installed packages changed. Likly they won't be able to completly uninstall, because dirs are not empty.